src/containers/BooksForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { createBook } from '../actions';
+
+class BooksForm extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      title: '',
+      category: ''
+    }
+    
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e){
+    const objName = e.target.name;
+    this.setState({
+      [objName]: e.target.value,
+    });
+  }
+
+  handleSubmit(e){
+    e.preventDefault();
+
+    this.props.createBook({
+      id: Math.floor((Math.random () * 50) + 1),
```

Comment:
Let's use a bigger range of ids as there is high chance of adding the same id to different books.
Or optionally could use a library like [shortid](https://www.npmjs.com/package/shortid) to generate ids.